service/src/main/java/com/ge/predix/acs/attribute/cache/RedisAttributeCache.java

```diff
@@ -0,0 +1,70 @@
+package com.ge.predix.acs.attribute.cache;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+import java.util.function.BiFunction;
+
+import org.apache.commons.lang3.StringUtils;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.data.redis.core.RedisTemplate;
+
+import com.ge.predix.acs.attribute.readers.CachedAttributes;
+
+public class RedisAttributeCache extends AbstractAttributeCache {
```

Comment:
can this be shared with the decision cache ? not sure if there is anything specific to attribute cache